src/components/approval-type/ApprovalType.vue

```diff
@@ -64,53 +105,86 @@ export default class ApprovalType extends Vue {
   /** Show only the court order option; remove via registrar option. */
   @Prop({ default: false }) readonly isCourtOrderOnly!: boolean
 
+  /** Draft notice date. */
+  @Prop({ default: '' }) readonly notice!: string
+
+  /** Draft application date. */
+  @Prop({ default: '' }) readonly application!: string
+
   // Local properties
   private courtOrderNumber = ''
   private courtOrderNumRules = []
   private valid = false
   private approvalTypeSelected = ''
+  private datePickerRules = []
+  private noticeDateText = ''
+  private applicationDateText = ''
 
   /** Called when component is mounted. */
   mounted (): void {
+    this.setDateRules()
+    this.setCourtNumberRules()
     // Copy props to mutable properties
     if (this.draftApprovedByRegistrar) {
```

Comment:
This only get run once, very early on -- possibly before the draft is fetched and parsed to store -- meaning any future update to the props is not handled by the component.

One way to handle this is to watch the props and set your local properties then.

Reply:
I still think this component has a issue where its local properties are not updated if the props change.